packages/fe-fpm-writer/src/page/types.ts

```diff
@@ -45,19 +106,24 @@ export interface CustomPage extends CustomElement {
     };
 }
 
+export interface FCL {
+    /**
+     * The FCL flag is calculated to true if the existing app is configured to use the flexible column layout
+     */
+    fcl?: boolean;
+
+    /**
+     * Represents the name of the control aggregation to be used for the custom page.
+     * This is only used if FCL is enabled and is calculated based on the optional sourcepage.
+     */
+    controlAggregation?: 'beginColumnPages' | 'midColumnPages' | 'endColumnPages';
```

Comment:
Looks like FCL is not. covered by any (snapshot) test

Reply:
I was looking for 'beginColumnPages' | 'midColumnPages' | 'endColumnPages' in the snapshots or tests and could not find any.

Reply:
That is indeed odd. Let me check.

Reply:
Check https://github.com/SAP/open-ux-tools/blob/fpm-op/packages/fe-fpm-writer/test/unit/page/__snapshots__/custom.test.ts.snap#L81

Reply:
OK